tools/functional-tester/etcd-tester/tester.go

```diff
@@ -115,3 +121,77 @@ func (s *Status) setCase(c int) {
 	defer s.mu.Unlock()
 	s.Case = c
 }
+
+// checkConsistency stops the cluster for a moment and get the hashes of KV storages.
+// TODO: instead of logging to stdout, record it as an event to database.
+func (c *cluster) checkConsistency() error {
+	log.Println("canceling all stressers")
+	for _, s := range c.Stressers {
+		s.Cancel()
+		defer s.Stress()
+	}
+
+	log.Println("sleeping...")
```

Comment:
log.Println("stopped stressing etcd cluster and waiting the cluster to fully recover...")